### PR TITLE
Changelog: Create cloud-noops-sql-db.md

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-07-25 at 1 58 19 pm

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-07-25 at 2 01 59 pm

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

**Embargo**

embargo

### CODE REVIEW

File: content/cloud/changelog/cloud-noops-sql-db.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+<img src="placeholder" alt="placeholder">
+
+<!-- break -->
+
+References:
+
+- [NoOps SQL Database (Private Beta) Documentation](/cloud/cloud-noops-sql-db.md)
+- [NOOps SQL Database (Private Beta) Blog Post](LINK-TBD)
+- [Discord Channel](LINK-TBD)
```
Comment (tpmccallum) [inline content/cloud/changelog/cloud-noops-sql-db.md:21]:
```suggestion
- [Discord Channel](https://www.fermyon.com/discord)
```

File: content/cloud/changelog/cloud-noops-sql-db.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+We're thrilled to announce Fermyon Cloud's NoOps SQLite Database is now in private beta. Spin developers can now persist relational data generated by their Spin application in a database that is provisioned and managed entirely by Fermyon Cloud on their behalf. No need to worry about database operations, connection strings, or resource management with Fermyon Cloud's NoOps SQLite Database; we've got you covered. We'd love to hear what you think of the experience. You can sign up for the private beta using [this short form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx). 
+
+<img src="placeholder" alt="placeholder">
+
+<!-- break -->
+
+References:
+
+- [NoOps SQL Database (Private Beta) Documentation](/cloud/cloud-noops-sql-db.md)
+- [NOOps SQL Database (Private Beta) Blog Post](LINK-TBD)
```
Comment (tpmccallum) [inline content/cloud/changelog/cloud-noops-sql-db.md:20]:
```suggestion
- [NOOps SQL Database (Private Beta) Blog Post](https://www.fermyon.com/blog/announcing-noops-sql-db)
```

File: content/cloud/changelog/cloud-noops-sql-db.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+title = "Fermyon Cloud NoOps SQL DB (Private Beta)"
+template = "changelog_item"
+date = "2022-07-26T16:00:00Z"
```
Comment (tpmccallum) [inline content/cloud/changelog/cloud-noops-sql-db.md:3]:
```suggestion
date = "2023-07-26T16:00:00Z"
```

File: content/cloud/changelog/cloud-noops-sql-db.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+enable_shortcodes = true
+tags = ["storage"]
+[extra]
+type= "changelog_post"
+
+---
+
+We're thrilled to announce Fermyon Cloud's NoOps SQLite Database is now in private beta. Spin developers can now persist relational data generated by their Spin application in a database that is provisioned and managed entirely by Fermyon Cloud on their behalf. No need to worry about database operations, connection strings, or resource management with Fermyon Cloud's NoOps SQLite Database; we've got you covered. We'd love to hear what you think of the experience. You can sign up for the private beta using [this short form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx). 
+
+<img src="placeholder" alt="placeholder">
```
Comment (tpmccallum) [inline content/cloud/changelog/cloud-noops-sql-db.md:13]:
```suggestion
```

**Review by tpmccallum — COMMENTED** [review]
Update year to 2023 and remove placeholder image.

**Review by macolso — APPROVED** [review]
LGTM - it would be nice to also add an image (assuming we can recycle the twitter card @flynnduism ?) but not blocking

**Pull request merged**